src/character.cpp

```diff
@@ -2654,31 +2654,19 @@ nc_color Character::symbol_color() const
 
     const auto &fields = g->m.field_at( pos() );
 
+    // Priority: electricity, fire, acid, gases
+    bool has_elec = false;
     bool has_fire = false;
     bool has_acid = false;
-    bool has_elec = false;
     bool has_fume = false;
     for( const auto &field : fields ) {
-        if( field.first == fd_incendiary || field.first == fd_fire ) {
-            has_fire = true;
-        }
```

Comment:
Maybe add `fd_fire_vent` and `fd_flame_burst` fields too?

Reply:
It was fixed in following linked PR.